addons/mail/static/src/core/common/thread_model.js

```diff
@@ -309,10 +305,6 @@ export class Thread extends Record {
         });
     }
 
-    get areAllMembersLoaded() {
```

Comment:
`fetchMembersState` still defined here in `thread_model.js` rather than the `discuss/` patch folder

Reply:
Same for `fetchMembersState`

Reply:
I've given up trying to move them all at once otherwise it's endless diff and conflicts.
I'm now happy with just moving whatever fits in my diff until we finally reach the goal, even if some fields moved to discuss are still used in mail temporarily.